packages/viz/src/index.js

```diff
@@ -38,6 +38,7 @@ import PhysicalTooltip from './components/daily/physicaltooltip/PhysicalTooltip'
 import ReservoirTooltip from './components/daily/reservoirtooltip/ReservoirTooltip';
 import ParameterTooltip from './components/daily/parametertooltip/ParameterTooltip';
 import ConfidentialTooltip from './components/daily/confidentialtooltip/ConfidentialTooltip';
+import WarmUpTooltip from './components/daily/warmup-tooltip/WarmUpTooltip';
```

Comment:
wondering why we have a dash there in the folder name while all the other folders do not have any "-"
```suggestion
import WarmUpTooltip from './components/daily/warmuptooltip/WarmUpTooltip';
```

Reply:
OK
Since `confidentialtooltip` is hard to read I think that one day I will rename all of them.